doc/development/style_guide.md

```diff
@@ -887,3 +943,81 @@ self.bwizard = wx.Button(...,
 # GTC %s will be replaced with name of current shell
 gs.message(_("Running through {}").format(shellname))
 ```
+
+### Developing C tools
+
+Refer to the [online GRASS Programmer's
+Manual](​https://grass.osgeo.org/programming8/) or generate it with `make
+htmldocs` or `make pdfdocs`.
+
+#### Use GRASS library functions
+
+Use the GRASS library functions instead of the standard C functions. The reason
+for this is that the following functions ensure good programming practice (e.g.
+always checking if memory was allocated) and/or improves portability.
```

Comment:
add "when available"

```suggestion
Use the GRASS library functions, when available, instead of the standard C
functions. The reason for this is that the following functions ensure good
programming practice (e.g. always checking if memory was allocated) and/or
improves portability.
```